README.md

```diff
@@ -393,6 +393,17 @@ The `all.json` file of the given hash. Other files, such as `missionRewards.json
 }
 ```
 
+## For Developers
+
+### Dependencies
+- [Node.js >= 14](https://nodejs.org/en/)
+
+### Usage
+You can run the build script with `npm run start`.
```

Comment:
this won't copy the built data, that's what `npm build` is for... also `run` is extraneous on `start`, since it's built-in

```suggestion
You can run the build script with `npm run build`.

```